app/models/concerns/mentionable.rb

```diff
@@ -48,20 +48,27 @@ def parse_id_from_decorated_ids(ids)
   end
 
   included do
-    def self.mentionable_users(opts = {})
-      users = active.exclude_user(User.current_user.id)
-                    .with_roles(mentionable_roles(opts[:roles])).distinct.to_a
-      users << add_order_creator(users, opts[:order_id]) if opts[:order_id]
+    def self.mentionable_users(roles:, order_id:, is_private: false)
+      users = User.active.exclude_user(User.current_user.id)
+                  .with_roles(mentionable_roles(roles)).distinct.to_a
+      users << add_order_creator(users, order_id) unless bool_cast.call(is_private)
```

Comment:
Looks good, but I would make it generic to any resource type, not just order. 
For sure in the coming months we'll talk about mentionable users on offers, etc